include/SFML/Graphics/Color.inl

```diff
@@ -89,7 +89,7 @@ constexpr Color operator-(const Color& left, const Color& right)
 {
     const auto clampedSub = [](std::uint8_t lhs, std::uint8_t rhs) -> std::uint8_t
     {
-        const int intResult = static_cast<int>(lhs) - static_cast<int>(rhs);
+        const int intResult = int{lhs} - int{rhs};
```

Comment:
I'd vote to keep the static_cast. It's clearer what's going on and I don't see the gain in saving a few characters.

Reply:
We gain the extra safety that a narrowing conversion isn't happening.

Reply:
Is it not? Looking at cppinsights, it just does a static_cast, so the only different is the difference in syntax, but maybe I misunderstand how cppinsights works 😅 

![image](https://user-images.githubusercontent.com/920861/210114400-b09abcc3-defa-4a29-8327-ca8b04dae838.png)

Reply:
I'm not entirely sure how cppinsights works but I can guarantee you that the two syntaxes are not identical and the braces are more restrictive in what they allow.